utopian-io/v2.utopian.io
# Add the Allow Externals field to the project form

### Verify that

---

### **Test plan**

_Allow project external contributions_

* Contribute a new project using the top nav bar: **Contribute**+ > **Add my project**
* Fill in the project's details
* Toggle the **Allow external contributions** to **on**

**Expected result:** project's allowExternals field is set to **true** in the database

_Do not allow project external contributions_

* Contribute a new project using the top nav bar: **Contribute**+ > **Add my project**
* Fill in the project's details
* Toggle the **Allow external contributions** to **off**

**Expected result:** project's allowExternals field is set to **false** in the database
